ironfish/src/rpc/routes/accounts/getBalance.ts

```diff
@@ -6,7 +6,7 @@ import { ApiNamespace, router } from '../router'
 import { getAccount } from './utils'
 
 export type GetBalanceRequest = { account?: string }
-export type GetBalanceResponse = { confirmed: string; unconfirmed: string }
+export type GetBalanceResponse = { accountName: string; confirmed: string; unconfirmed: string }
```

Comment:
maybe this is bad on our part, but when we accept an account name we use the variable `account` like above on line 8. Can you make this consistent for now so at least they don't diverge until we decide to make a change here?

Reply:
makes sense. updated.

Reply:
this change applies to PR #1086 too. will update accordingly.